CIP-xxxxx/README.md

```diff
@@ -0,0 +1,218 @@
+---
+CIP: ???
+Title: Web-Wallet Bridge - Plutus based wallets
+Status: Proposed
+Category: Wallets
+Authors: 
+  - Leo
+Implementors: NA
+Discussions:
+Created: 2023-10-12
+License: CC-BY-4.0
+---
+
+
+## Abstract
+
+This document describes a CIP-30 extension allowing webpages (i.e. dApps) to interface with Cardano Plutus-wallets. This document is a work in progress and is not yet finalized. It is expected to be updated as the ecosystem evolves. 
```

Comment:
```suggestion
This document describes a CIP-30 extension allowing webpages (i.e. dApps) to interface with Cardano Plutus wallets. This interface is a work in progress and new versions are expected to be included as the ecosystem evolves. 
```
beyond the "Plutus wallets" suggestion... the original text gives us no grounds for merging or activating this CIP.  Does this revision reflect what you state below in the Versioning section?